### ISSUE TITLE
Error with method call on require

### ISSUE THREAD

**Issue by Eschon** [lcodes/esdoc-node#5]
Calling a method directly on the require doesn't work.
Here's an example:

```
require('dotenv').config();
```

When running esdoc with this plugin it logs the following error:

```
warning: could not parse the following code. if you want to use ECMAScript proposals, see https://esdoc.org/manual/feature.html#ecmascript-proposal
/Users/egon/src/dolphin/test.js
1| require('dotenv').config();
2|
```

**Comment by eliasthompson** [comment]
Running into this exact scenario as well, as well as on things like `require('./sample')(router);`. Any plan to address this?

**Comment by eliasthompson** [comment]
Created a PR (#6) that would add this functionality for you to look over @lcodes.

**Comment by lcodes** [comment]
Merged and published as 1.0.4. Thanks!
